src/js/claims-status/components/appeals-v2/Docket.jsx

```diff
@@ -1,10 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-class Docket extends React.Component {
-  render() {
-    return (null);
-  }
+function Docket({ ahead, total }) {
+  const completedWidth = { width: `${((total - ahead) / total) * 100}%` };
+
+  // TODO: Assess how accessible this is
+  return (
+    <div className="docket-container">
+      <span className="appeals-ahead">{ahead.toLocaleString()}</span>
+      <p>Appeals ahead of you</p>
+
+      <div className="marker-container">
+        <div>
+          <div className="marker-text-spacer" style={completedWidth}/>
+          <span className="marker-text">You are here</span>
+        </div>
+        <div>
+          <div className="spacer" style={completedWidth}/>
+          <div className="marker">
+            <img src="/img/Docket-line-pin.svg" alt=""/>
```

Comment:
No alt is intentional?

Reply:
Yup. It'll make more sense to a screen reader to ignore the whole progress bar and marker, methinks, but we'll address that in a separate ticket.